lib/rex/powershell/command.rb

```diff
@@ -15,14 +15,14 @@ module Command
   # @option opts [Bool] :sub_funcs Substitute function names
   #
   # @return [String] Encoded script
-  def self.encode_script(script_in, opts={})
+  def self.encode_script(script_in, eof=nil, opts={})
```

Comment:
Since this changes the calling syntax, are you sure that all other callers to this method have been updated?